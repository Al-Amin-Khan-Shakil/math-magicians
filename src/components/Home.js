const Home = () => (
  <div className="home-wrapper">
    <h2 className="home-title">
      Experience Math Like Never Before with Math Magicians!
    </h2>
    <p className="home-paragraph">
      Welcome to Math Magicians, where numbers come to life, equations dance at your
      fingertips, and math becomes an enchanting journey. Our cutting-edge calculator app
      is your passport to a world where math isn&apos;t just about solving problems; it&apos;s about
      exploring the wonders of the numerical universe.
    </p>
    <p className="home-paragraph">
      In this digital realm, math transcends the mundane and transforms into an art form.
      With Math Magicians, you&apos;ll discover that math is not just a tool for daily tasks;
      it&apos;s a gateway to uncover the beauty and elegance hidden within each equation.
    </p>
  </div>
);

export default Home;
