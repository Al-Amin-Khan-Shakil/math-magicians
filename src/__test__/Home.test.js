import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('Test the Home Component', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
