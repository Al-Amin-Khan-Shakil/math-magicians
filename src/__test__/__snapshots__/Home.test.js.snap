// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Home Component 1`] = `
<div
  className="home-wrapper"
>
  <h1
    className="home-title"
  >
    Experience Math Like Never Before with Math Magicians!
  </h1>
  <p
    className="home-paragraph"
  >
    Welcome to Math Magicians, where numbers come to life, equations dance at your fingertips, and math becomes an enchanting journey. Our cutting-edge calculator app is your passport to a world where math isn't just about solving problems; it's about exploring the wonders of the numerical universe.
  </p>
  <p
    className="home-paragraph"
  >
    In this digital realm, math transcends the mundane and transforms into an art form. With Math Magicians, you'll discover that math is not just a tool for daily tasks; it's a gateway to uncover the beauty and elegance hidden within each equation.
  </p>
</div>
`;
